templates/skeleton/app/routes/pages/$pageHandle.tsx

```diff
@@ -32,19 +32,54 @@ const seo: SeoHandleFunction<typeof loader> = ({data}) => ({
   description: data?.page?.seo?.description,
 });
 
+// wait what's going on here??
 export const handle = {
   seo,
 };
 
 export const meta: MetaFunction = ({data}) => {
-  const {title, description} = data?.page.seo ?? {};
+  try {
+    const {title, description} = data?.page.seo ?? {};
 
-  return {
-    title,
-    description,
-  };
+    return {
+      title,
+      description,
+    };
+  } catch (error) {
+    console.error(error);
+    return {};
+  }
 };
 
+export function ErrorBoundary({error}: {error: unknown}) {
+  return (
+    <div>
+      There was an error!
```

Comment:
Maybe to further illustrate the scope of the boundary, we could add `There was an error on this page` instead - because it's a `page` route